spring-cloud-gcp-logging/src/main/java/org/springframework/cloud/gcp/logging/StackdriverJsonLayout.java

```diff
@@ -0,0 +1,83 @@
+/*
+ *  Copyright 2018 original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.springframework.cloud.gcp.logging;
+
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.classic.spi.IThrowableProxy;
+import com.google.cloud.logging.TraceLoggingEnhancer;
+import com.google.gson.Gson;
+
+/**
+ * @author Andreas Berger
+ */
+public class StackdriverJsonLayout extends ch.qos.logback.contrib.json.classic.JsonLayout {
+
+	public static final String SEVERITY_ATTR_NAME = "severity";
```

Comment:
I have never liked abbreviation, especially public API.

We develop here Spring library, so would be great to be as friendly for users as possible.
`SEVERITY_ATTRIBUTE` or even just `SEVERITY` with good JavaDocs is fully enough.
But, please, no abbreviations.

Reply:
BTW, would be great to have JavaDocs on all the public API, including constructors and constants.